sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -85,6 +85,11 @@ export class ChatClient {
     const authPolicy = createCommunicationTokenCredentialPolicy(this.tokenCredential);
     this.client.pipeline.addPolicy(authPolicy);
 
+    (options as any).signalingClientOptions = {
+      ...(options as any).signalingClientOptions,
+      resourceEndpoint: this.endpoint
```

Comment:
what happened if resourceEndpoint of signaling client options are provided? Should we override it?

Reply:
SignalingClientOptions does not accept resourceEndpoint in the past, so doing so would cause a compile error. So endpoint URL should not be part of signalingClientOptions